crates/autopilot/src/driver_api.rs

```diff
@@ -1,5 +1,5 @@
 use {
-    crate::driver_model::{execute, solve},
+    crate::driver_model::{settle, solve},
```

Comment:
Slightly unrelated, but I noticed that this name is wrong and had to fix it. There is no `/execute`.